src/reporter.test.ts

```diff
@@ -362,6 +361,7 @@ describe('reporter', () => {
         },
         { dummyEnd },
       ];
+      const shorter = 'a'.repeat(1944);
```

Comment:
do we not have a constant for the max length?

Reply:
Yes but it is maddening to calculate backwards what the size oughta be in the test. In reality, the red flag is if the leftover characters change in an unrelated PR.